pkg/tasks/telemeter.go

```diff
@@ -278,6 +278,7 @@ func (t *TelemeterClientTask) destroy(ctx context.Context) error {
 	return errors.Wrap(err, "creating Telemeter Client serving certs CA Bundle ConfigMap failed")
 }
 
+//nolint:unused
```

Comment:
I think we can create a "technical debt" issue to get rid of this, IIUC it's no longer using after https://github.com/openshift/cluster-monitoring-operator/commit/23356b2df280acacfb8132616f3abe3e98ef2dbc and I don't think the function can/is being used by any other package.

Reply:
why not remove it in this PR then?